docs/DocumentationStyleGuide/accessibility.rst

```diff
@@ -0,0 +1,94 @@
+Accessibility
+=============
+
+The moja global community strives towards accessibility in our software
+products and even in the documentation. Through accessibility, we strive
+for successful access to information and resources among people who
+might find it challenging to access them otherwise. We aim to bring
+contributors with limited ability to the forefront of our community to
+have them contribute and access our documentation easily.
+
+The general guidelines for creating accessible documentation pieces are:
+
+-  Maintain consistency across the documentation. Both in style and
+   visuals by employing a fixed hierarchy.
+-  Avoid language or word usage that reeks of bias or a disregard for a
+   group of people.
+-  Focus on the audience that you are writing for and follow a
+   conversation style of documentation.
+-  Define acronyms, abbreviations, prerequisites initially and never
+   assume the technical competency of the reader.
+-  Use alternative text for multimedia elements wherever possible, and
+   be conscious of the usage of visual elements.
+
+Image accessibility
+-------------------
+
+-  Validate that the images, diagrams, graphs, icons possess a
+   meaningful alternative-text description for a meaningful equivalent.
+-  Use alternative text to describe what the image, digraph, graph or
+   icon does, rather than what it looks like.
+-  All images, diagrams, graphs, icons should contain a short image
+   description that is self-explanatory.
+-  Prefer the usage of SVGs over JPEGs and PNGs. SVGs scale on zooming
+   and provide better context to the readers.
+-  Don’t use images of text, terminal outputs, code samples and other
+   information that can be displayed as text.
+
+Text accessibility
+------------------
+
+-  Follow a content hierarchy defined by the content strategy or the
+   documentation plan for the project.
+-  Use Heading-1 only once in the entire content piece. Use Heading-2
+   and Heading-3 for all the sub-headings (#, ##, ### in Markdown).
+-  Headings should be correctly nested to indicate their relative
+   significance in the documentation piece.
+-  Avoid Camel-Case, inconsistent formatting and unnecessary
+   capitalization of text.
+-  Always break down your documentation piece into short paragraphs that
```

Comment:
Can we suggest using Tables of Contents wherever possible?

It's useful to tell readers what they're about to read in a condensed form. TOC achieve this, but the same principle can be replicated at the start of each new section.

I firmly believe that readers are lazy. We need to make it as easy as possible to access the desired understanding.